pom.xml

```diff
@@ -171,24 +170,33 @@
                     <version>${maven-install-plugin.version}</version>
                 </plugin>
                 <plugin>
-                    <groupId>net.revelc.code.formatter</groupId>
-                    <artifactId>formatter-maven-plugin</artifactId>
-                    <version>${formatter-maven-plugin.version}</version>
+                    <groupId>com.diffplug.spotless</groupId>
+                    <artifactId>spotless-maven-plugin</artifactId>
+                    <version>${spotless-maven-plugin.version}</version>
                     <configuration>
-                        <cachedir>.cache</cachedir>
-                    </configuration>
-                </plugin>
-                <plugin>
-                    <groupId>net.revelc.code</groupId>
-                    <artifactId>impsort-maven-plugin</artifactId>
-                    <version>${impsort-maven-plugin.version}</version>
-                    <configuration>
-                        <cachedir>.cache</cachedir>
-                        <groups>java.,javax.,org.,io.,com.</groups>
-                        <staticGroups>*</staticGroups>
-                        <staticAfter>true</staticAfter>
-                        <removeUnused>true</removeUnused>
+                        <java>
+                            <includes>
+                                <include>src/main/java/**/*.java</include>
+                                <include>src/test/java/**/*.java</include>
+                            </includes>
+                            <eclipse>
+                                <file>ide-config/eclipse-format.xml</file>
+                            </eclipse>
+                            <importOrder>
+                                <file>ide-config/eclipse.importorder</file>
+                            </importOrder>
+                            <removeUnusedImports/>
+                        </java>
                     </configuration>
+                    <executions>
```

Comment:
Based on https://github.com/diffplug/spotless/tree/main/plugin-maven#binding-to-maven-phase, but somehow this doesn't work for me. Also without this configuration no `spotless:check` is executed when I run `mvn verify` (which should be the default).

Reply:
see it here: https://github.com/operator-framework/java-operator-sdk/blob/0c20e6b6726a483dbdb2c116089f02bbe0dc5bf9/pom.xml#L325-L355

Reply:
I've aligned the config with `java-operator-sdk`, also kept the XML files in the `contributing` folder.